eitprocessing/datahandling/loading/__init__.py

```diff
@@ -1,13 +1,13 @@
 from functools import reduce
 from pathlib import Path
 
-from eitprocessing.data_collection import DataCollection
-from eitprocessing.eit_data import EITData
-from eitprocessing.eit_data.vendor import Vendor
-from eitprocessing.sequence import Sequence
+from eitprocessing.datahandling.datacollection import DataCollection
+from eitprocessing.datahandling.eitdata import EITData, Vendor
+from eitprocessing.datahandling.loading import draeger, sentec
+from eitprocessing.datahandling.sequence import Sequence
 
 
-def load_data(
+def load_eit_data(
```

Comment:
@psomhorst, why was this change made? I thought because we are automatically loading all attached data (continuous and sparse), we decided to call it `load_data` without specifying eit

Reply:
Ah, yes, true. But we will extend `loading` with functions loading data from different sources, e.g. from a mechanical ventilator or external pressure sensors. `load_data()` would then become ambiguous.